models/src/main/scala/coop/rchain/models/ParMap.scala

```diff
@@ -15,11 +15,12 @@ case class ParMap(
 ) {
 
   override def equals(o: scala.Any): Boolean = o match {
-    case parMap: ParMap => this.ps == parMap.ps && this.remainder == parMap.remainder
-    case _              => false
+    case parMap: ParMap =>
+      this.ps == parMap.ps && this.remainder == parMap.remainder && this.connectiveUsed == parMap.connectiveUsed
```

Comment:
> If connectiveUsed were different it means we have a bug.

Agreed. Then it makes even more sense to make the equals consistent with other variants of the AST, because comparing it can lead to exposing a bug. Ignoring it can't.